dart/lib/src/protocol/sentry_user.dart

```diff
@@ -37,13 +39,21 @@ class SentryUser {
     this.email,
     this.ipAddress,
     this.segment,
-    Map<String, dynamic>? extras,
-  })  : assert(id != null ||
-            username != null ||
-            email != null ||
-            ipAddress != null ||
-            segment != null),
-        extras = extras == null ? null : Map.from(extras);
+    this.subscription,
+    this.geo,
+    this.name,
+    Map<String, dynamic>? data,
+    @Deprecated('Will be removed in v7. Use [data] instead')
+        Map<String, dynamic>? extras,
+  })  : assert(
+          id != null ||
+              username != null ||
+              email != null ||
+              ipAddress != null ||
+              segment != null,
+        ),
+        assert(data == null || extras == null, 'Only use one of data/extra'),
+        data = (data ?? extras) == null ? null : Map.from(data ?? extras ?? {});
```

Comment:
This is a breaking change in behavior, I'd rather keep both for now, without fallbacks, just as it is, but adding `data`.
And later just remove `extras`.